configure.py

```diff
@@ -740,10 +740,8 @@ def check_compiler(o):
   ok, is_clang, clang_version, gcc_version = try_check_compiler(CXX, 'c++')
   if not ok:
     warn('failed to autodetect C++ compiler version (CXX=%s)' % CXX)
-  elif sys.platform.startswith('aix') and gcc_version < (6, 3, 0):
-    warn('C++ compiler too old, need g++ 6.3.0 (CXX=%s)' % CXX)
-  elif clang_version < (3, 4, 2) if is_clang else gcc_version < (4, 9, 4):
-    warn('C++ compiler too old, need g++ 4.9.4 or clang++ 3.4.2 (CXX=%s)' % CXX)
+  elif clang_version < (3, 4, 2) if is_clang else gcc_version < (6, 3, 0):
```

Comment:
IIUC https://en.wikipedia.org/wiki/Xcode#Latest_versions then:
```suggestion
  elif clang_version < (8, 0, 0) if is_clang else gcc_version < (6, 3, 0):
    warn('C++ compiler too old, need g++ 6.3.0 or clang++ 8.0.0 (CXX=%s)' % CXX)
```

Reply:
👍 if https://github.com/nodejs/node/pull/26714/files#r266236173 is resolved (either removed or bumped).

Reply:
@refack I bumped the clang versions to 8.0.0. PTAL.

Reply:
Seems like [we might need `9.0.0` for mac](https://github.com/nodejs/node/pull/26714#discussion_r268011985), and ??? for real llvm.

Reply:
P.S. ??? could be 8.0.0 now that https://codesmithdev.com/llvm-8-has-been-released-with-new-toolchain-requirements/

Reply:
I'll land this as 8.0.0 (once I'm satisfied I haven't broken Travis in another [PR](https://github.com/nodejs/node/pull/26720#issuecomment-475465044)). We can alway change it again in another PR. Theses are only advisory warnings, afterall.